src/content/migrate/4.md

```diff
@@ -156,3 +161,10 @@ module.exports = {
 ## module.loaders
 
 `module.loaders` were deprecated since webpack 2 and are now removed in favor of [`module.rules`](/configuration/module/#rule).
+
+
+## Additional Reading
```

Comment:
Put this into the file header header in related section: https://webpack.js.org/contribute/writers-guide#yaml-frontmatter

Reply:
@Kolhar730 this is still a request, please put into related section, e.g. 
```
---
title: My Article
group: My Sub-Section
sort: 3
contributors:
  - [github username]
related:
  - title: Title of Related Article
    url: [url of related article]
---
```